room.cpp

```diff
@@ -163,9 +163,17 @@ void Room::setJoinState(JoinState state)
     emit joinStateChanged(oldState, state);
 }
 
+void Room::setLastReadEvent(User* user, QString eventId)
```

Comment:
We can't move it to `Room::Private` because it's used from `QuaternionRoom`. But it's a good idea to make it protected, I agree.